docs/ecosystem_release/starrocks-kubernetes-operator.md

```diff
@@ -0,0 +1,163 @@
+
+Go: 1.19 or later
+
+## **Release note**
+
+## 1.8
+
+**1.8.6**
+
+**Bug Fixes**
+
+Fixed the following issue:
+
+- An error  `sendfile() failed (32: Broken pipe) while sending request to upstream` is return during a Stream Load job. After Nginx sends the request body to FE, FE then redirects the request to BE. At this point, the data cached in Nginx may already be lost. [#303](https://github.com/StarRocks/starrocks-kubernetes-operator/pull/303)
+
+**Doc**
+
+- [Load data from outside the K8s network to StarRocks through FE proxy](https://github.com/StarRocks/starrocks-kubernetes-operator/blob/main/doc/load_data_using_stream_load_howto.md)
+- [Update the root user's password using Helm](https://github.com/StarRocks/starrocks-kubernetes-operator/blob/main/doc/change_root_password_howto.md)
+
+**1.8.5**
+
+**Improvements**
+
+- **[Helm Chart] The `annotations` and `labels` can be customized for the service account of the operator**: The Operator creates a service account named `starrocks` by default, and users can customize the annotations and labels for the service account `starrocks` of the operator by specifying the `annotations` and `labels` fields in `serviceAccount` in **values.yaml**. The `operator.global.rbac.serviceAccountName` field is deprecated. [#291](https://github.com/StarRocks/starrocks-kubernetes-operator/pull/291)
+- **[Operator] FE service supports explicit protocol selection for Istio**: When Istio is installed in the Kubernetes environment, Istio needs to determine the protocol of the traffic from the StarRocks cluster, in order to provide additional functionality such as routing and rich metrics. So FE service explicitly defines its protocol as MySQL in the `appProtocol` field. This improvement is particularly important because the MySQL protocol is a server-first protocol that is incompatible with automatic protocol detection and sometimes may incur connection failures. [#288](https://github.com/StarRocks/starrocks-kubernetes-operator/pull/288)
+
+**Bug Fixes**
+
+- **[Helm Chart]** The root user's password in StarRocks may not be initialized successfully when `starrocks.initPassword.enabled` is true and the value of `starrocks.starrocksCluster.name` is specified. It is caused by the wrong FE service domain name used by the initpwd pod to connect FE service. More specifically, in this scenario, FE service domain name uses the value specified in `starrocks.starrocksCluster.name`, while the initpwd pod still uses the value of the `starrocks.nameOverride` field to form the FE service domain name. ([#292](https://github.com/StarRocks/starrocks-kubernetes-operator/pull/292))
+
+**Upgrade notes**
+
+- **[Helm Chart]** When the value specified in `starrocks.starrocksCluster.name` is different from the value of `starrocks.nameOverride`, the old configmaps for FE, BE, and CN will be deleted. New configmaps with the new name for FE, BE, and CN will be created. **This may result in the restart of FE/BE/CN pods.**
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/216b4849-4eeb-4311-a5bd-d9d92bee48bf)
-> with new names for the FE, BE, and CN
-> the FE, BE, and CN pods